cpp/src/Ice/Exception.cpp

```diff
@@ -47,7 +47,7 @@
 #        endif
 #    endif
 
-#    if !defined(__FreeBSD__) && defined(ICE_API_EXPORTS)
```

Comment:
Not sure if we can remove this `defined(ICE_API_EXPORTS)`. It seems to work fine on macos. Would be good to test on Linux with a static build.